templates/data-sources/users.md.tmlp

```diff
@@ -0,0 +1,16 @@
+---
+page_title: "authentik_users Data Source - terraform-provider-authentik"
+subcategory: "Directory"
+description: |-
+  Get users list
+---
+
```

Comment:
```suggestion

# {{.Name}} ({{.Type}})

{{ .Description | trimspace }}

{{ if .HasExample -}}
## Example Usage

{{ printf "{{tffile %q}}" .ExampleFile }}
{{- end }}

{{ .SchemaMarkdown | trimspace }}
```

Reply:
I pushed the requested changes. However I am surprised because the other files in `templates/data-sources` are not templated like this.